ui/raidboss/data/07-dt/raid/r4n.ts

```diff
@@ -21,21 +22,46 @@ const effectB9AMap = {
 type B9AMapKeys = keyof typeof effectB9AMap;
 type B9AMapValues = typeof effectB9AMap[B9AMapKeys];
 
+const directionOutputStrings = {
+  dirN: Outputs.dirN,
+  dirE: Outputs.dirE,
+  dirS: Outputs.dirS,
+  dirW: Outputs.dirW,
+  goLeft: Outputs.left,
+  goRight: Outputs.right,
+  unknown: Outputs.unknown,
```

Comment:
```suggestion
  ...Directions.outputStringsCardinalDir,
  goLeft: Outputs.left,
  goRight: Outputs.right,
```
I think `outputStringsCardinalDir` includes all 4 and `unknown`.